File: Domasna_2/prototip/stocktopus-frontend/src/model/dto/stockDetailsDTO.ts
export interface StockDetailsDTO {
    detailsId: number;
    stockId: number;
    stockName: string;
    date: Date;
    lastTransactionPrice: string;
    maxPrice: string;
    minPrice: string;
    averagePrice: string;
    percentageChange: string;
    quantity: string;
    tradeVolume: string;
    totalVolume: string;
}

File: Domasna_2/prototip/stocktopus-backend/src/main/java/mk/finki/ukim/mk/stocktopusbackend/model/dto/StockDetailsDTO.java
package mk.finki.ukim.mk.stocktopusbackend.model.dto;

import java.util.Date;

public record StockDetailsDTO(
        String id,
        Long stockId,
        String stockName,
        Date date,
        String lastTransactionPrice,
        String maxPrice,
        String minPrice,
        String averagePrice,
        String percentageChange,
        String quantity,
        String tradeVolume,
        String totalVolume
) {}
